trino-s3-proxy/src/main/java/io/trino/s3/proxy/server/remote/AwsRemoteS3FacadeConstants.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.s3.proxy.server.remote;
+
+public final class AwsRemoteS3FacadeConstants
+{
+    private static final String AWS_DOMAIN = "amazonaws.com";
+    public static final RemoteS3HostBuilder VIRTUAL_HOST_BUILDER = ((bucket, region) -> (bucket.isEmpty() ? "" : "%s.".formatted(bucket)) + "s3.%s.%s".formatted(region, AWS_DOMAIN));
```

Comment:
I find the () around the whole lambda hard on the eyes. It's not needed. Do you mind removing?